tests/report_tests/plugin_tests/systemd.py

```diff
@@ -0,0 +1,91 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos_tests import StageOneReportTest, StageTwoReportTest
+
+
+class SystemdPluginTest(StageOneReportTest):
+    """Basic sanity check to make sure common config files are collected
+
+    :avocado: tags=stageone
+    """
+
+    sos_cmd = '-o systemd'
+
+    def test_systemd_files_collected(self):
+        self.assertFileGlobInArchive("/etc/systemd")
+        self.assertFileGlobInArchive("/lib/systemd/system")
+        self.assertFileGlobInArchive("/lib/systemd/user")
+        self.assertFileGlobInArchive("/etc/vconsole.conf")
```

Comment:
The `Glob` test is redundantly generic for a direct filename without a glob, but let it be..